.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--- Please provide a general summary of your changes in the Title above -->
+
+# PR Description
+
+<!--- Describe the changes your PR makes in detail -->
+
+References issue: # (ISSUE)
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## Tests and Coverage
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, tests ran to see how your change affects other areas of the code, etc. -->
+
+## Types of changes
+
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Docs (Added to or improved Databay's documentation)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+## Final Checklist:
```

Comment:
I'd maybe suggest adding something about code coverage here too. For example:

```
[ ] I've run coverage on the project
The coverage is: [INPUT COVERAGE PERCENTAGE HERE]
```

We could also add how to run coverage (and tests) into CONTRIBUTING, and link to it here.

Reply:
Nice suggestion adding that to the PR